pkg/runit/servicebuilder.go

```diff
@@ -48,6 +49,12 @@ const (
 // trailing space must be used.
 const yamlSeparator = "--- "
 
+// This matches the pods package. Copied to avoid import cycles
```

Comment:
Could `pkg/pods` just import `pkg/runit` and use `runit.DefaultLogExec`? I don't see any imports pointing in the other direction right now